doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -254,6 +254,8 @@ In addition to this, custom logging also allows the following fields:
 
 * "certificate": The TLS certificate base64 encoded
 * "chain": The entire TLS certificate chain base64 encoded
+* "ja3_hash": The JA3 hash, which is an md5sum of the JA3 string
```

Comment:
a m5sum instead of an md5sum?

Reply:
What is a 'm5sum'? Isn't 'md5sum' correct here?